src/python/update_container_definitions.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import
 import sys
 import json
+import re
```

Comment:
Is this import still required?

Reply:
No I don't think so thank you for this reminder